src/sentry/api/helpers/constants.py

```diff
@@ -0,0 +1,2 @@
+MAX_QUERY_SUBSCTIPTIONS_HEADER = "X-Sentry-Rule-Limit"
```

Comment:
`SUBSCTIPTIONS` typo 😛 also should this be `X-Sentry-Alert-Rule-Limit`?

Reply:
at least i was consistent in my typo. 🤣 - ty - updating!

for `X-Sentry-Alert-Rule-Limit` do you know if the snuba subscription limit only impacts metric alerts? (i believe so, in which case agree that this is a better name)

Reply:
Yeah, if you look [here](https://github.com/getsentry/sentry/blob/master/src/sentry/incidents/serializers/alert_rule.py#L470-L482) it's checking the `QuerySubscription`s which are only used for metric alerts.